src/lib/orders.ts

```diff
@@ -87,6 +88,20 @@ export const isMarketOrderType = (type?: AbacusOrderTypes) =>
     AbacusOrderType.trailingStop,
   ].some(({ ordinal }) => ordinal === type.ordinal);
 
+export const isStopLossOrder = (order: SubaccountOrder) =>
+  [AbacusOrderType.stopLimit, AbacusOrderType.stopMarket].some(
+    ({ ordinal }) => ordinal === order.type.ordinal
+  ) &&
+  order.reduceOnly &&
+  order.timeInForce?.name === TimeInForceOptions.IOC;
```

Comment:
QQ: Why not TimeInForceOptions.FOK as well?

Reply:
hmm good q - I think it's because, implicitly, when a user creates a SL/TP order through the new figma flow, we set it to IOC (and there's no option to customize it), but maybe that's not a great reason. Lemme double check with Se to verify if we wanna include FOK orders, and maybe allow users to customize it in the new flow

Reply:
if a user places a sl/tp order via the old trade sidebar flow and have execution other than IOC, will we show that in the table?

Reply:
chatted with Se (some context [here](https://www.figma.com/file/fMQodZKzn9B5aZTN5G0fLJ?node-id=24617:2257&mode=design#723145235) as well); since all our orders created through the edit modal will be implicitly IOC (and there's no ability to change that), we will only show the explicit order in the table if it's IOC (and not FOK). If it is FOK, we'll instead show this "View orders" [text](https://www.figma.com/file/fMQodZKzn9B5aZTN5G0fLJ/dYdX-%E2%80%BA-Desktop?type=design&node-id=24233-7151&mode=design&t=oQZMM0DOpLjk1PiZ-0) (I know it says Multiple, but designs are being updated there) which will take the user to the "Orders" tab where they can delete and remake the order if they prefer. Reasoning behind this is that we don't show IOC/FOK in the edit modal. 

That being said - will update this PR to reflect this behavior!

Reply:
updated!